app/frontend/src/pages/CreditsPage/_CreditsPage.scss

```diff
@@ -0,0 +1,126 @@
+@use "components/Basics" as *;
+
+.credits-intro-container {
+  background-image: url("assets/images/svgs/credits-page-bg-top.svg");
+  background-position: top;
+  background-repeat: no-repeat;
+  background-size: 100%;
+  height: auto;
+  width: 100%;
+}
+
+.body-container {
+  background-image: url("assets/images/svgs/credits-page-bg-top.svg");
+  background-position: top center;
+  background-repeat: no-repeat;
+  background-size: 100%;
+  padding: 5% 10%;
+}
+
+.join-us-section {
+  background-image: url("assets/images/svgs/credits-page-bg-bottom.svg");
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center bottom;
+  position: relative;
+  height: 600px;
+}
+
+.join-us-content-container {
+  position: relative;
+  top: 250px;
+  justify-content: center;
+  align-items: center;
+}
+
+.btn-outline {
+  background-color: #fff;
+  color: $color-blue-dark;
+  font-weight: 400;
+}
+
+.btn-border {
+  border: 1px solid #3450a1;
+}
+
+.btn-outline:hover {
+  color: #fff;
+  font-weight: bold;
+}
+
+.text-container {
+  margin-bottom: 25%;
+  @include breakpoint-media-max("smtablet") {
+    margin-bottom: 10%;
+  }
+}
+
+.credits-intro-title {
+  width: 45%;
+
+  @include breakpoint-media-max("smtablet") {
+    margin-top: 20%;
+  }
+}
+
+.credits-intro-paragraph {
+  width: 45%;
+  @include breakpoint-media-max("smtablet") {
+    margin-top: 10%;
+    width: 100%;
+  }
+}
+
+.main-container {
+  max-width: 1088px;
+  margin: 0 auto;
+}
+
+.credits-test {
+  color: $color-charcoal;
+}
+
+.grid {
+  display: grid;
+  max-width: 1088px;
+  margin: 0 auto;
+  grid-template-columns: repeat(auto-fit, minmax(292px, 1fr));
+  gap: 5rem;
+  margin-top: 5rem;
+}
```

Comment:
Just a note: it's probably not as big of a problem as it was a couple years ago, but it looks like grid is not supported by IE (this is one reason why most design system uses flexbox to make grids, like mui, bootstrap, or carbon).

I think, if we want to support grid, we should include it as a layout rather than a page specific class. Thoughts?

Reply:
(I read more on IE support for Grid and think flexbox might be a better choice for now. I'm working on changing the layout to flexbox. We can keep discussing whether grid classes are needed and I'm happy to work on those if needed.)

Reply:
I'm on team flexbox for the simplicity but also may be biased as I don't have much experience with grid. 

Definitely open to discussion about why @bzzz-coding prefers grid!